wgpu-core/src/hub.rs

```diff
@@ -80,6 +80,9 @@ impl IdentityManager {
 enum Element<T> {
     Vacant,
     Occupied(T, Epoch),
+    #[cfg(debug_assertions)]
+    Error(Epoch, String),
```

Comment:
I think we can carry `String` unconditionally here. If it's empty, it doesn't cost much, and the `Occupied` already takes more size, so adding `String` to `Error` variant wouldn't change the total size of the `enum`.